frontend/src/__tests__/cypress/cypress/pages/distributedWorkloads.ts

```diff
@@ -29,6 +30,21 @@ class GlobalDistributedWorkloads {
     return cy.findByTestId('project-selector-dropdown');
   }
 
+  findRefreshIntervalSelectToggle() {
+    return cy.get('#metrics-toolbar-refresh-interval-select-toggle');
+  }
+
+  selectRefreshInterval(interval: RefreshIntervalTitle) {
+    this.findRefreshIntervalSelectToggle().click();
+    cy.findByTestId('metrics-toolbar-refresh-interval-select').within(() => {
+      cy.findByText(interval).click();
+    });
```

Comment:
Can you use `findSelectOption` for this?
Also, I believe you can just chain the selectors without using `within`.
```suggestion
    cy.findByTestId('metrics-toolbar-refresh-interval-select').findSelectOption(interval).click();
```

Reply:
ah I always forget that exists, thanks.